airbyte-api/src/main/openapi/config.yaml

```diff
@@ -972,6 +1014,22 @@ paths:
           description: Connection not found
         "422":
           $ref: "#/components/responses/InvalidInput"
+  /v1/operations/list_unassigned:
```

Comment:
Are you going to implement it for UI needs or for some other use-cases? I do not think that we are going to use it on UI according our current mockups.

Reply:
let's remove it